kong/db/strategies/postgres/init.lua

```diff
@@ -1409,22 +1428,22 @@ function _M.new(connector, schema, errors)
     insert_statement = concat {
       "INSERT INTO ",  table_name_escaped, " (", insert_columns, ")\n",
       "     VALUES (", insert_expressions, ")\n",
-      "  RETURNING ",  select_expressions, ";",
+      "  RETURNING ", select_expressions, ";",
     }
 
     upsert_statement = concat {
       "INSERT INTO ",  table_name_escaped, " (", insert_columns, ")\n",
       "     VALUES (", insert_expressions, ")\n",
       "ON CONFLICT (", pk_escaped, ") DO UPDATE\n",
       "        SET ",  upsert_expressions, "\n",
-      "  RETURNING ",  select_expressions, ";",
+      "  RETURNING ", select_expressions, ";",
     }
 
     update_statement = concat {
       "   UPDATE ",  table_name_escaped, "\n",
       "      SET ",  update_expressions, "\n",
       "    WHERE (", pk_escaped, ") = (", update_placeholders, ")\n",
-      "RETURNING ",  select_expressions , ";"
+      "RETURNING ", select_expressions, ";",
```

Comment:
I don't think we want these stylistic changes?

Reply:
We have no strict rule for that. On the one hand improving the code as you modify it is considered a good practice. On the other hand, ideally style changes should be on their own commit (with a `style` namespace). On the third hand, these changes were so small that adding an extra commit for them feels overkill.